instance.tf

```diff
@@ -0,0 +1,74 @@
+resource "aws_instance" "public" {
+  ami                         = "ami-0912f71e06545ad88"
+  associate_public_ip_address = true
+  instance_type               = "t2.micro"
+  key_name                    = "dropmailtokishan"
```

Comment:
Could you please rename the key to `main` it will then be more easier for me to troubleshoot the code from my side if needed